requirements.txt

```diff
@@ -0,0 +1,7 @@
+colorlog==6.7.0
+pip>=21.0,<23.2
+ruff==0.0.265
+
+# Install HA and test dependencies (pytest, coverage)
+# See https://github.com/MatthewFlamm/pytest-homeassistant-custom-component/releases for version mappings
+pytest-homeassistant-custom-component==0.13.32 # HA 2023.5.4
```

Comment:
Does this pin our dependencies to a precise Home Assistant version?

Reply:
Yes, in this case to 2023.5.4. This only affects the dev container and can be easily changed to any other version by updating the dependency version and rebuilding the container. I clarified this in 93ddaa0.

We could remove the version pinning so that the latest version is installed, but for a reproducible dev environment it might be advantageous if it's pinned?

Reply:
On the one hand I see an argument for keeping it pinned, however, because Home Assistant changes quite quickly leaving it unpinned might be the right thing.

Reply:
You're right, it only affects the dev container anyway. 5bc30457.